sympy/core/numbers.py

```diff
@@ -1021,6 +1027,8 @@ def __new__(cls, num, dps=None, prec=None, precision=None):
 
         if isinstance(num, float):
             _mpf_ = mlib.from_float(num, precision, rnd)
+        elif type(num).__module__ == 'numpy':
+            _mpf_ = mlib.from_float(num, precision, rnd)
```

Comment:
Eventually we probably want to do this.  But I'm not sure np floats work in all possible paths through the above code for dealing with all the special cases for precision and dps.  For example, does `Float(np...., '')` work?

Reply:
Will remove if you insist but my suggestion is to keep it allows us to do things like
```
Float(np.float128(5),12)
Float(np.float32(5.1200000),dps=5)
```
Which is better than nothing. The example, you gave indeed fails, but so does `Float(<int>, '')` so it is a more general issue.

Reply:
I didn't say remove.  I hope the other PR makes those things work.

I assume you mean `Float(<numpy.int>, '')`: please file bug if `Float<sympy int>, '')` isn't working.